plugins/permanentpen.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: default
+title: Permanent Pen Plugin
+title_nav: Permanent Pen
+description: Apply formats while typing
+keywords: permanent pen copy text format style
+controls: toolbar button, contextmenu, menu item
+---
+
+The Permanent Pen allows a user to apply a pre-defined format (color, typeface and type style, or font) to a desired section of content overriding the default values set for that content area.
+
+The Permanent Pen is useful when a distinctly styled text needs to be repeatedly added to the content. The user can pre-define the desired style in the Permanent Pen properties. This pre-defined style is applied to all the text entered into the document with the Permanent Pen enabled. The ability to store information for reuse saves the user time and ensures consistency across the content.
```

Comment:
"...when a distinctly styled text..." -> "...when a distinct style..." ? And do we want to use just "format" or just "style" or use both?

"...entered into the document with the Permanent Pen enabled." -> "...entered into the document while the Permanent Pen is enabled."?